test/e2e/aws.go

```diff
@@ -0,0 +1,52 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type awsTestHelper struct {
```

Comment:
Will simplify it. But looks the credentials don't have enough permission either. 

`    operator_test.go:475: failed to create aws wafv2 acl due to operation error WAFV2: CreateWebACL, https response error StatusCode: 400, RequestID: aca0d697-3801-422d-9d3e-b67d3b6bff52, api error AccessDeniedException: User: arn:aws:iam::460538899914:user/origin-ci-robot-provision is not authorized to perform: wafv2:CreateWebACL on resource: arn:aws:wafv2:us-west-2:460538899914:regional/webacl/echoserver-acl/* because no identity-based policy allows the wafv2:CreateWebACL action`

and 

`     operator_test.go:575: Ensuring test workload namespace aws-load-balancer-test-wafv2
    operator_test.go:598: failed to get change token for waf classic operation error WAF: GetChangeToken, https response error StatusCode: 400, RequestID: 30dc3a79-2347-4e34-9b60-61e4225d2a0c, api error AccessDeniedException: User: arn:aws:iam::460538899914:user/origin-ci-robot-provision is not authorized to perform: waf:GetChangeToken on resource: arn:aws:waf::460538899914:changetoken/* because no identity-based policy allows the waf:GetChangeToken action `